src/data/chainDetails.json

```diff
@@ -799,5 +835,158 @@
 			"url": "https://raw.githubusercontent.com/spothq/cryptocurrency-icons/master/svg/color/eth.svg",
 			"format": "svg"
 		}
+	},
+	"97": {
+		"name": "BNB Smart Chain Testnet",
+		"chain": "BSC",
+		"rpc": [
+			"https://data-seed-prebsc-1-s1.bnbchain.org:8545",
+			"https://data-seed-prebsc-2-s1.bnbchain.org:8545",
+			"https://data-seed-prebsc-1-s2.bnbchain.org:8545",
+			"https://data-seed-prebsc-2-s2.bnbchain.org:8545",
+			"https://data-seed-prebsc-1-s3.bnbchain.org:8545",
+			"https://data-seed-prebsc-2-s3.bnbchain.org:8545",
+			"https://bsc-testnet.publicnode.com"
+		],
+		"faucets": [
+			"https://testnet.bnbchain.org/faucet-smart"
+		],
+		"nativeCurrency": {
+			"name": "BNB Chain Native Token",
+			"symbol": "tBNB",
+			"decimals": 18
+		},
+		"infoURL": "https://www.bnbchain.org/en",
+		"shortName": "bnbt",
+		"chainId": 97,
+		"networkId": 97,
+		"slip44": 1,
+		"explorers": [
+			{
+				"name": "bscscan-testnet",
+				"url": "https://testnet.bscscan.com",
+				"standard": "EIP3091"
+			}
+		],
+		"mainnet": false,
+		"icon": {
+			"url": "https://raw.githubusercontent.com/trustwallet/assets/8ee07e9d791bec6c3ada3cfac73ddfdc4f4a40b7/blockchains/bnbt/info/logo.png",
+			"format": "png"
+		}
+	},
+	"43113": {
```

Comment:
Same, is this a newly deployed chain? If so, pls communicate so, have to make sure this is active in UI too (and api aswel)

Reply:
arbitrum testnet, don't bother adding to ui